alnair-device-plugin/build/upload.sh

```diff
@@ -0,0 +1,2 @@
+sudo docker build -f Dockerfile -t centaurusinfra/alnair-device-plugin:ann .
```

Comment:
add the user to the docker group instead of using sudo? also the tag needs to be consistent with our release?
Not sure this script is needed for now.